packages/admin/addon/templates/admin.hbs

```diff
@@ -1,2 +1,27 @@
 {{! Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
-{{outlet}}
+<div class="navi-admin">
+    <DenaliNavbar class="navi-admin__navbar" @isResponsive={{true}} as |Nav|>
+        <Nav.Left as |Left|>
+            <Left.Icon @icon="menu-dialpad" {{on "click" (toggle "isTabExpanded" this)}} />
```

Comment:
I wouldn't worry about this now. We need to finalize the nav bar. This most likely will follow the pattern here: 

https://denali.design/docs/2/guides/get-started

Responsive and hamburger menu for small screens.